src/components/CalendarPicker/index.js

```diff
@@ -0,0 +1,158 @@
+import _ from 'underscore';
+import React from 'react';
+import {View, TouchableOpacity} from 'react-native';
+import moment from 'moment';
+import Text from '../Text';
+import ListPicker from './ListPicker';
+import ArrowIcon from './ArrowIcon';
+import styles from '../../styles/styles';
+import {propTypes, defaultProps} from './calendarPickerPropTypes';
+import generateMonthMatrix from './generateMonthMatrix';
+
+const monthNames = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+const daysOfWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+
+const addMonths = (date, months) => {
+    const d = new Date(date.getTime());
+    const newMonth = d.getMonth() + months;
+    d.setMonth(newMonth);
+    return d;
+};
```

Comment:
i think we could refactor the code and operate only on moment instead of this function